config/environments/production.rb

```diff
@@ -6,6 +6,8 @@
   config.action_mailer.default_options = { from: "hello@getyourrefund.org" }
   config.address_for_transactional_authentication_emails = 'no-reply@getyourrefund.org'
   config.action_mailer.default_url_options = { host: 'www.getyourrefund.org' }
+  config.action_mailer.asset_host = "https://getyourrefund.org"
```

Comment:
I think `https://getyourrefund.org/*` redirects to `https://www.getyourrefund.org/*`, based on visiting this URL https://getyourrefund.org/assets/checkbox-logo--white-252f6970859e7b1ba37367e7820af3472b45415066bb501fae76813011fde48e.png and seeing it redirect. It's nicer if we don't redirect people, so I'll probably submit a trivial PR to make this say `www.`

It's also not a blocker.